openssl/src/ocsp.rs

```diff
@@ -345,3 +374,48 @@ foreign_type_and_impl_send_sync! {
     pub struct OcspOneReq;
     pub struct OcspOneReqRef;
 }
+
+pub fn check_nonce(req: &OcspRequestRef, bs: &OcspBasicResponseRef) -> Result<(), ErrorStack> {
+    unsafe {
+        cvt(ffi::OCSP_check_nonce(req.as_ptr(), bs.as_ptr()))?;
```

Comment:
This function will need a custom return type, looking at the documentation.

Reply:
I've implemented this custom return. I don't *love* what I've done. Please let me know what you think.